### PR TITLE
Step 4 change (Using C++ and Pybind)

### PR DESCRIPTION
**Pull request by kliu3638**
image

### CODE REVIEW

**Review by minnce — CHANGES_REQUESTED** [review]
speed is good, just rebase so it can merge

**Comment by echavemann** [comment]
rebase is not the problem

**Comment by echavemann** [comment]
just need to resolve merge conflict

**Comment by echavemann** [comment]
@minnce we need to improve PR quality this is an issue

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "configurations": [
+        {
```
Comment (echavemann) [inline .vscode/c_cpp_properties.json:3]: please ignore .vscode

File: cpp/primeFactor.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#include "primeFactor.h"
+#include <iostream>
+#include <vector>
+#include <unordered_map>
+#include <unordered_set>
```
Comment (echavemann) [inline cpp/primeFactor.cpp:5]: clean up unused imports

File: prime.py [hunk 3]
```diff
@@ -1,22 +1,37 @@
 import timeit
+import PrimeFactor
+
+def disasterCode(upper = 1000):
+    primeFactor = PrimeFactor.PrimeFactor()
+    primeFactor.calcPrimes(upper)
+
+    # # Sieve to get primes
```
Comment (echavemann) [inline prime.py:8]: delete commented code

**Review by echavemann — CHANGES_REQUESTED** [review]

**Comment by kliu3638** [comment]
Made changes accordingly.

**Review by minnce — APPROVED** [review]
looks good to me now!

**Pull request merged**